### ISSUE TITLE
Unable to read in local file for rainfall dataset 

### ISSUE THREAD

**Issue by cdeciampa** [tropycal/tropycal#141]
Not a big deal, but line 51 in rain/dataset.py should be changed to `self.rain_df = pd.read_csv(data_path)`. It's currently hard-coded to only read in the WPC rainfall dataset.

<img width="541" alt="image" src="https://user-images.githubusercontent.com/66640080/145694108-b832b36e-8bd8-41c4-b60f-0326cf71d685.png">

**Comment by tomerburg** [comment]
Thanks for noticing! I’ll fix this with the next bug fix. The idea was to make the WPC dataset default while allowing for the option of reading in other datasets if they exist and/or are available; do you know of any in addition to WPC by any chance?

**Comment by cdeciampa** [comment]
No, I don’t know of any other datasets similar in structure to the WPC dataset. That one was made in 2008 using the WPC's CLIQR database and [tweaked last summer](https://www.wpc.ncep.noaa.gov/tropical/rain/milestones.html) by a student intern at WPC, so the formatting is unique to WPC.

I ended up making my own csv with an identical structure to this dataset using ASOS data cross referenced against IBTrACS for a final project last semester (to include 2021's rainfall), but its purpose was very specific to my project and would need a lot of work before it could be made public. But thank you for making tropycal! I used this package extensively in my project (with proper citation, of course).
